src/service/MatchmakingService.ts

```diff
@@ -69,6 +71,19 @@ export class MatchmakingService extends APIClient {
         return await this.invokeApi(`/matches/estates/${estateId}/count`, 'GET');
     }
 
+    /**
+     * Fetch matches for a given searchprofile
+     * @param searchpofileId
+     * @param page
+     * @param size
+     * @param offset
+     */
+    async fetchMatchesBySearchprofile(searchprofileId: string, page: number = 0, size: number = 10, offset: number = 0): Promise<AxiosResponse<PagedResult<SearchprofileMatch>>> {
+        return await this.invokeApi(`/matches/searchprofile/${searchprofileId}`, 'GET', undefined, {
```

Comment:
Bitte den Typ in den Call rein geben, der Rückgabewert ist dann entsprechend automatisch

```suggestion
    async fetchMatchesBySearchprofile(searchprofileId: string, page: number = 0, size: number = 10, offset: number = 0) {
        return await this.invokeApi<PagedResult<SearchprofileMatch>>(`/matches/searchprofile/${searchprofileId}`, 'GET', undefined, {
```

Reply:
Außerdem wäre `invokeApiWithErrorHandling<PagedResult<SearchprofileMatch>>` noch besser